src/Language/Haskell/Liquid/Types.hs

```diff
@@ -458,8 +458,9 @@ instance Hashable (PVar a) where
 ------ Strictness --------------------------------------------------
 --------------------------------------------------------------------
 
-instance NFData Var
-instance NFData SrcSpan
+instance NFData Var where rnf _ = ()
```

Comment:
I forgot about this, the new version of `NFData` requires a `Generic` instance for automatic derivation, but GHC doesn't supply them for most types :disappointed: